File: Results/DER/Translation/translation_output/misleading_test/codellama/codenet/python/java-sanitized/p02885.java
import java.util.Scanner;

public class p02885 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        int diff = A - B * 2;
        if (diff < 0) {
            System.out.println(0);
        } else {
            System.out.println(diff);
        }
    }
}



12 4


59

File: Results/DER/Translation/translation_output/w_test/granite/codenet/java/python/p02885.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
12 4

Expected output:
4


```
import java.util.*;

public class p02885 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String line = sc.nextLine();
        String[] line1 = line.split(" ");
        int a = Integer.parseInt(line1[0]);
        int b = Integer.parseInt(line1[1]);
        if(a > 2 * b)   System.out.println(a - 2 * b);
        else System.out.println(0);
    }
}
```


Answer:
```python
import sys

def main():
    line = sys.stdin.readline()
    a, b = map(int, line.split())
    if a > 2 * b:
        print(a - 2 * b)
    else:
        print(0)

if __name__ == "__main__":
    main()
```

Test input:
```
12 4
```

Expected output:
```
4
```<|endoftext|>
